packages/ra-ui-materialui/src/auth/Logout.tsx

```diff
@@ -35,7 +34,7 @@ const LogoutWithRef: FunctionComponent<
     Props & MenuItemProps<'li', { button: true }> // HACK: https://github.com/mui-org/material-ui/issues/16245
 > = React.forwardRef(function Logout(props, ref) {
     const { className, redirectTo, ...rest } = props;
-    const classes = useStyles();
+    const classes = useStyles({});
```

Comment:
I remember doing that in my mui upgrade PR. As this is a temporary workaround, you should add a link to the github issue

Reply:
Done. I just don't understand why TypeScript complains in these 2 instances of `useStyles()` and not in the many other ones in the codebase.